extensions/amp-analytics/0.1/events.js

```diff
@@ -16,6 +16,9 @@
 
 import {CommonSignals} from '../../../src/common-signals';
 import {Observable} from '../../../src/observable';
+import {PlayingStates} from '../../../src/service/video-manager-impl';
```

Comment:
@cvializ dependency check doesn't like this, let's move `PlayingStates` enum to `video-interface`